.github/dependabot.yml

```diff
@@ -7,4 +7,4 @@ updates:
 - package-ecosystem: "nuget"
   directory: "/"
   schedule:
-    interval: "weekly"
+    interval: "weekly"
```

Comment:
Could you please, setup editorconfig recognition in the IDE and revert such changes?